include/common/libfsm_common.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 Scott Vokes
+ *
+ * See LICENCE for the full copyright terms.
+ */
+
+#ifndef LIBFSM_COMMON_H
+#define LIBFSM_COMMON_H
```

Comment:
would you mind naming this something like check.h please? that way it doesn't invite more miscellaneous stuff in the same header

Reply:
Updated in 2f154973.